packages/core/test/financial-templates/expiring-multiparty/PricelessPositionManager.js

```diff
@@ -2111,8 +2203,25 @@ contract("PricelessPositionManager", function(accounts) {
     assert.equal(collateralPaid, toWei("60"));
 
     // Push a different price to the new oracle to ensure the contract still uses the old price.
```

Comment:
Is this test done also for liquidation+dispute price requests? i.e the Dispute resolves to a certain price, and then OO is switched out in the finder, but the Perp should still settle the dispute and pay out rewards using the originally resolved dispute price.

Reply:
Good point. Will make sure this is tested.

Reply:
Wait, sorry, I don't think we _can_ test this. Because the settlement for liquidation and disputes happens at the same time as the price retrieval. So you pull the price and immediately send the payouts. There is no in-between time for you to swap out the oracle. Does that make sense?

Reply:
Yup, I thought that might be the case. Nice